integration-tests/rpc-nodes.spec.ts

```diff
@@ -433,6 +444,18 @@ CONFIGS().forEach(
           expect(paidSpace).toBeDefined();
           done();
         });
+
+        MumbaiAndAlpha('Verify that rpcClient.ticketBalance will retrieve the specified ticket owned by the given contract', async (done) => {
+          const ticketBalance = await rpcClient.getTicketBalance(ticketContract.address, { ticketer: ticketContract.address, content_type: { prim: 'string' }, content: { string: 'abc' } } );
+          expect(ticketBalance).toBeDefined();
+          done();
+        });
+  
+        MumbaiAndAlpha('Verify that rpcClient.allTicketBalances will retrieve all tickets owned by the given contract', async (done) => {
+          const ticketBalances = await rpcClient.getAllTicketBalances(ticketContract.address!);
```

Comment:
this doesnt give a typescript error? i thought we didnt allow non-null assertions

Reply:
Don't know why it didn't complaint, but good catch thank you!! It's removed now